runtime/manta/src/lib.rs

```diff
@@ -794,6 +800,8 @@ impl pallet_collective::Config<TechnicalCollective> for Runtime {
     type MaxProposals = ConstU32<100>;
     type MaxMembers = ConstU32<100>;
     type DefaultVote = pallet_collective::PrimeDefaultVote;
+    type SetMembersOrigin = EnsureRoot<AccountId>;
```

Comment:
We need to do a sudo call on-chain to set the Council and Technical committee

Reply:
also we need to change the safe xcm version to 3 with governance/sudo on both chains

Reply:
Exactly!